src/webob/acceptparse.py

```diff
@@ -1859,131 +5102,3 @@ def fdel(request):
             pass
 
     return property(fget, fset, fdel, textwrap.dedent(doc))
-
-
-class MIMEAccept(Accept):
```

Comment:
We should keep this around, even if it is just a shim that calls the new classes/methods. `MIMEAccept` is pretty heavily used by various other libraries/tools that depend on WebOb.